projects/ngrx.io/content/guide/effects/lifecycle.md

```diff
@@ -24,7 +24,7 @@ Usage:
 
 <code-example header="log.effects.ts">
 import { Injectable } from '@angular/core';
-import { Actions, Effect, ofType } from '@ngrx/effects';
+import { Actions, createEffect, ofType } from '@ngrx/effects';
```

Comment:
this was just a driveby change that I noticed the import was wrong

Reply:
Could you create a separate PR for this change please?

Reply:
sure

Reply:
moved to https://github.com/ngrx/platform/pull/2336